aea/crypto/ledger_apis.py

```diff
@@ -112,7 +112,7 @@ def token_balance(self, identifier: str, address: str) -> int:
             except Exception:
                 logger.warning("An error occurred while attempting to get the current balance.")
                 balance = 0
-        else:           # pragma: no cover
+        else:       # pragma: no cover
             balance = 0
```

Comment:
Question: in the case when the Ledger id is not known, shouldn't this raise an exception instead? Or return `None` (in this case, change the return value to `Optional[int]` in the signature).

Reply:
+1 for exception!